sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisResult.java

```diff
@@ -46,9 +45,9 @@
 
     private final Map<String, List<SensitivityValue>> valuesByContingencyId = new HashMap<>();
 
-    private final Map<Triple<String, String, String>, SensitivityValue> valuesByContingencyIdAndFunctionIdAndVariableId = new HashMap<>();
+    private final EnumMap<SensitivityFunctionType, Map<Triple<String, String, String>, SensitivityValue>> valuesByContingencyIdAndFunctionIdAndVariableId = new EnumMap<>(SensitivityFunctionType.class);
```

Comment:
Maybe we should have a dedicated object for that, because it starts to be hard to understand like SensitivityValueKey and FunctionReferenceKey ?